src/dotnet-svcutil/lib/src/FrameworkFork/Microsoft.Xml/External/Extensions.cs

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+using System.IO;
+using System.Reflection;
+
+namespace Microsoft.Xml {
+				using System;
+				
+    public static class Extensions
+    {
+        public static void Close(this Stream stream)
+        {
+            stream.Dispose();
+        }
+
+        public static MethodInfo GetMethod(this Type type, String name, BindingFlags bindingAttr, System.Reflection.Binder binder, CallingConventions callConvention, Type[] types, ParameterModifier[] modifiers)
+        {
+            return type.GetMethod(name, types);
+        }
+
+        public static MethodInfo GetMethod(this Type type, String name, BindingFlags bindingAttr, System.Reflection.Binder binder, Type[] types, ParameterModifier[] modifiers)
+        {
+            return GetMethod(type, name, bindingAttr, binder, CallingConventions.Standard, types, modifiers);
+        }
+    }
+}
+
+
+namespace Microsoft.Xml.Serialization {
+				using System;
+				using Microsoft.Xml;
+    public static class SerializationExtensions
+    {
+        public static TypeCode GetTypeCode(this Type type)
+        {
```

Comment:
I thought this method was brought back in later netstandard versions so this isn't needed?